fn.go

```diff
@@ -2,6 +2,9 @@ package main
 
 import (
 	"context"
+	"fmt"
+
+	"github.com/antonmedv/expr"
```

Comment:
Any thoughts on `expr` vs [CEL](https://github.com/google/cel-spec)? I don't know either well, but I know CEL is used in Kubernetes (e.g. for CRD validation). It might be a little more familiar to our audience.

Reply:
CC @ytsarev and @avalanche123, who worked on `function-if` that inspired this.

Reply:
I would be good opening another PR to support CEL instead.

Reply:
@negz I don't have any reservations agains either. from hackathon implementations velocity, expr seemed easier. cel seems like the way to go for k8s/protobuf ecosystem.

Reply:
> I would be good opening another PR to support CEL instead.

@stevendborrelli Yeah, please do.

Reply:
Using CEL is a great idea 👍